File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int hight;
    do
    {
        hight = get_int("Height: ");
    }
    while (hight <= 0 || hight >= 9);



    for (int i = 1 ; i <= hight ; i++)
    {
        int space = hight - i;
        while (space)
        {
            printf(" ");
            space--;
        }
        for (int j = 1 ; j <= i ; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while 1:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(1,n+1):
    print(' '*(n-i) + '#'*i + ' '*2 + '#'*i)
